lib/dash/content_protection.js

```diff
@@ -320,6 +320,50 @@ shaka.dash.ContentProtection = class {
     return ContentProtection.getLaurl_(rootElement);
   }
 
+
+  /**
+   * Gets a PlayReady initData from a content protection element
+   * containing a PlayReady Pro Object
+   *
+   * @param {shaka.dash.ContentProtection.Element} element
+   * @return {?Array.<shaka.extern.InitDataOverride>}
+   */
+  static getInitDataFromPro(element) {
+    const proNode = shaka.util.XmlUtils.findChildNS(
+        element.node, 'urn:microsoft:playready', 'pro');
+    if (!proNode) {
+      return null;
+    }
+    const fromBase64 = shaka.util.Uint8ArrayUtils.fromBase64;
```

Comment:
You should only alias the type, not the function directly.  Calling the function as a "raw" function causes problems with `this`.  It would be better to:

```js
const Uint8ArrayUtils = shaka.util.Uint8ArrayUtils;
const foo = Uint8ArrayUtils.fromBase64(...);
```